frontend/src/app/App.tsx

```diff
@@ -9,11 +9,19 @@ import Routes from './Routes';
 import NavSidebar from './NavSidebar';
 
 import './App.scss';
+import { useHistory } from 'react-router';
+import { useSegmentIOTracking } from '../utilities/useSegmentIOTracking';
+import { fireTrackingEvent, initSegment } from '../utilities/segmentIOUtils';
+import { RootState } from '../redux/types';
```

Comment:
nit: as convention, we import outer modules first (react-router), then local modules, then CSS. Can't find a lint rule to support that though.